examples/ftp-service-send-file/ftp_service_send_file.metatags

```diff
@@ -1,2 +1,2 @@
-description: This BBE is about receiving file/directory changes that occur in a remote FTP server and using the `ftp:Caller` to interact with the FTP server.
+description: This example demonstrates receiving file/directory changes that occur in a remote FTP server and using an ftp:Caller to interact with the FTP server.
```

Comment:
Same confusion here, the description does not say about appending content to the added files. May be we can improve the two descriptions

Reply:
Fixed